plugins/kubernetes-react/package.json

```diff
@@ -63,9 +63,11 @@
     "@kubernetes-models/apimachinery": "^2.0.0",
     "@kubernetes-models/base": "^5.0.0",
     "@kubernetes/client-node": "1.0.0-rc7",
-    "@material-ui/core": "^4.9.13",
-    "@material-ui/icons": "^4.11.3",
-    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@material-ui/core": "^4.12.2",
```

Comment:
is this required still?

Reply:
sadly, I had to revert some changes; you can see this [commit](https://github.com/backstage/backstage/pull/27572/commits/07a5180dbe302aa93f88fcff6ccbea9c511c8677) because some tests were failing with `TypeError: theme.spacing is not a function`, it can be seen [here](https://github.com/backstage/backstage/actions/runs/11766038722/job/32773123569)